src/locales/am/common.json

```diff
@@ -245,7 +245,7 @@
     "sendMax": "ሁሉንም ይላኩ",
     "settleMore": "ጨምረው ይወስኑ",
     "settleTransactions": "ግብይቱን ይወስኑ",
-    "shareAddress": "አድራሻ ይጋሩ",
+    "shareAddress": ".ር ያድርጉ",
```

Comment:
How did this happen? 
https://translate.google.com/?sl=auto&tl=en&text=.%E1%88%AD%20%E1%8B%AB%E1%8B%B5%E1%88%AD%E1%8C%89&op=translate

Reply:
https://user-images.githubusercontent.com/66996635/120495841-b0b6f800-c3da-11eb-9cf7-e546ded4847b.mov

Reply:
@mm319 - what should it _actually_ be?